apps/coscrad-frontend/src/components/NavBar/NavBarItem.spec.tsx

```diff
@@ -0,0 +1,20 @@
+import { MemoryRouter } from 'react-router-dom';
+import { renderWithProviders } from '../../utils/test-utils';
+import { NavItemInfo } from './NavBar';
+import { NavBarItem } from './NavBarItem';
+
+const dummyNavBarItemProps: NavItemInfo = {
+    link: 'http://samplesite.com',
+    label: 'Sample Label',
+};
+
+describe('NavBarItem', () => {
```

Comment:
I'm not too worried about having a dedicated test for this. There's not much that can go wrong, but anything that does go wrong should be caught by the higher integration test. That said, there's no harm in keeping it around. Thanks for keeping testing a priority!

Reply:
sounds good.